File: LeetCode/Python/jewelsAndStones.py
class Solution(object):
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        ans = 0
        jewel = set()
        for j in J:
            jewel.add(j)

        for s in S:
            if s in jewel:
                ans += 1

        return ans

File: LeetCode/CSharp/JewelsAndStones.cs
public class Solution {
    public int NumJewelsInStones(string J, string S) {
        int ans = 0;
        var jewel = new HashSet<char>();

        for(int i = 0; i < J.Length; i++)
        {
            jewel.Add(J[i]);
        }

        for(int i = 0; i < S.Length; i++)
        {
            if(jewel.Contains(S[i])) ans++;
        }
        
        return ans;
    }
}